pkg/test/resourcefixture/testdata/basic/alloydb/v1beta1/alloydbinstance/fullalloydbinstance/update.yaml

```diff
@@ -30,3 +30,17 @@ spec:
     enablePublicIp: false
     enableOutboundPublicIp: false
     authorizedExternalNetworks: []
+  queryInsightsConfig:
```

Comment:
Could you actually update some fields you added? This is the update step of the test case and we want to support the update scenarios properly.

Reply:
> Could you actually update some fields you added? This is the update step of the test case and we want to support the update scenarios properly.

i have updated the fullalloydbinstance test with multiple fields changing, in a single request, logs: https://paste.googleplex.com/4664440502091776